src/mutations/member.ts

```diff
@@ -146,10 +146,14 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
       const newFavoriteItems = prevExtra.favoriteItems
         ? prevExtra.favoriteItems.concat([itemId])
```

Comment:
Another way is to only concat if it does not exist:
```suggestion
        ? prevExtra.favoriteItems.concat(prevExtra.favoriteItems.includes(itemId) ? [] : [itemId])
```